tests/MongoDB.Driver.Core.Tests/Core/Operations/GroupOperationTests.cs

```diff
@@ -405,7 +405,7 @@ public void CreateCommand_should_return_expected_result_when_MaxTime_is_set(long
             [Values(false, true)]
             bool async)
         {
-            RequireServer.Check().Supports(Feature.Collation);
+            RequireServer.Check().Supports(Feature.GroupCommand).Supports(Feature.Collation);
```

Comment:
Supports accepts multiple features. You can use:

    ...Suports(Feature.GroupCommand, Feature.Collation);